modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/frontend/data/set/action/AdvancedFDSItemsActions.java

```diff
@@ -32,12 +39,101 @@ public class AdvancedFDSItemsActions implements FDSItemsActions {
+		sidePanel1FDSActionDropdownItem.putData(
+			"title", "Side Panel Title Provided by Action");
+
+		FDSActionDropdownItem sidePanel2FDSActionDropdownItem =
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createRenderURL(
+					liferayPortletResponse
+				).setMVCRenderCommandName(
+					"/side_panel/full"
+				).setWindowState(
+					LiferayWindowState.POP_UP
+				).buildString(),
+				"rectangle-split", "open-side-panel-title",
+				"Side Panel With Action and Content Title", null, null,
+				"sidePanel");
+
+		sidePanel2FDSActionDropdownItem.putData("disableHeader", false);
+		sidePanel2FDSActionDropdownItem.putData(
+			"title", "Side Panel Title Provided by Action");
+
+		FDSActionDropdownItem sidePanel3FDSActionDropdownItem =
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createRenderURL(
+					liferayPortletResponse
+				).setMVCRenderCommandName(
+					"/side_panel/full"
+				).setWindowState(
+					LiferayWindowState.POP_UP
+				).buildString(),
+				"rectangle-split", "open-side-panel-title",
+				"Side Panel With Content Title", null, null, "sidePanel");
+
+		sidePanel3FDSActionDropdownItem.putData("disableHeader", true);
+
 		return Arrays.asList(
 			new FDSActionDropdownItem(
-				_language.get(httpServletRequest, "are-you-sure"), "danger",
-				null, "#", "home", "navigateHome",
-				_language.get(httpServletRequest, "nav-link"), null, null,
-				"get", null, null, "link", null, "item"));
+				null, "view", "sampleMessage", "Sample View", null, null, null),
+			new FDSActionDropdownItem(
+				"#test-pencil", "pencil", "sampleEditMessage", "Sample Edit",
+				null, null, null),
+			new FDSActionDropdownItem(
+				"#test-delete", "times-circle", "sampleDeleteMessage",
+				"Sample Delete", null, null, null),
+			new FDSActionDropdownItem(
+				"#test-copy", "copy", "sampleMoveFolderMessage", "Sample Copy",
+				null, null, null),
+			new FDSActionDropdownItem(
+				href, "truck", "asyncSuccess", "Async Success", "get", null,
+				"async"),
+			new FDSActionDropdownItem(
+				"http://localhost", "times-circle",
+				"asyncErrorConnectionRefused", "Async Connection Refused",
+				"get", null, "async"),
+			sidePanel1FDSActionDropdownItem, sidePanel2FDSActionDropdownItem,
+			sidePanel3FDSActionDropdownItem,
+			new FDSActionDropdownItem(
+				PortletURLBuilder.createRenderURL(
+					liferayPortletResponse
+				).setMVCRenderCommandName(
+					"/side_panel/empty"
+				).setWindowState(
+					LiferayWindowState.POP_UP
+				).buildString(),
+				"rectangle-split", "open-side-panel-without-title",
+				"Side Panel With No Title", null, null, "sidePanel"),
+			new FDSActionDropdownItem(
+				href + "/abc", "staging", "asyncErrorResourceNotFound",
+				"Async Resource Not Found", "get", null, "async"),
+			new FDSActionDropdownItem(
+				null, "reload", "reload", "Reload Data", null, null, null),
+			new FDSActionDropdownItem(
+				null, "rectangle-split", "openSidePanel", "Open Side Panel",
+				null, null, null));
```

Comment:
I noticed the ["Import a system data set to customize"](https://testray.liferay.com/#/project/35392/routines/45357/build/209504336/case-result/209506117) PW test is failing because `target` is required so that parameter can't be kept `null` as it was in the existing code.

This part of the test will also need to be updated to reflect the different items https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-admin-web/main/tests/data-set-admin/systemDataSets.spec.ts#L280-L290

(Let me know if you'd like me to make these changes though)

Reply:
thanks a lot @thektan , I'm adding a commit to fix it. Also, added the portlet name so that import process (made from `FDSAdminPortlet` respects the original portlet key when creating the action URLs

Reply:
@thektan About the part of the test in  https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-admin-web/main/tests/data-set-admin/systemDataSets.spec.ts#L280-L290, please go ahead and fix it, thanks a lot!

Reply:
I'll fix this text as part of https://github.com/liferay-frontend/liferay-portal/pull/4887